test/extended/testdata/testjob-plugin.xml

```diff
@@ -53,16 +69,62 @@
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
 
+    <!-- 1 to 1 mapping -->
     <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
       <apiURL>https://openshift.default.svc.cluster.local</apiURL>
       <namespace>PROJECT_NAME</namespace>
+      <testStream>origin-nodejs-sample</testStream>
       <testTag>latest</testTag>
+      <prodStream>origin-nodejs-sample</prodStream>
       <prodTag>prod</prodTag>
+      <authToken></authToken>
+    </com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+
+    <!-- 1 to N mapping -->
+    <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <namespace>PROJECT_NAME</namespace>
       <testStream>origin-nodejs-sample</testStream>
+      <testTag>latest</testTag>
       <prodStream>origin-nodejs-sample</prodStream>
+      <prodTag>prod2,prod3,prod4</prodTag>
```

Comment:
@bparees since I unfortunately originally named the instance variables testTag, prodTag, etc. we have a backwards compatibility issue, with folks at older versions of the plugin potentially having job config's with that old naming convention.

whether that unfortunately reality needs to bleed into new tests we create is certainly a subject for debate.... I tend to agree that we should test with the preferred names.